tests/platform_tests/test_link_down.py

```diff
@@ -158,20 +164,33 @@ def test_link_down_on_sup_reboot(duthosts, localhost, enum_supervisor_dut_hostna
     # RP doesn't have any interfaces, check all LCs' interfaces
     link_status_on_all_LC(duthosts, localhost, fanouts_and_ports, up=False)
 
-    time.sleep(MAX_TIME_TO_REBOOT)
+    # Wait for ssh port to open up on the DUT
+    wait_for_startup(duthost, localhost, 0, MAX_TIME_TO_REBOOT)
 
     dut_uptime = duthost.get_up_time()
     logger.info('DUT {} up since {}'.format(hostname, dut_uptime))
     rebooted = float(dut_uptime_before.strftime("%s")) != float(dut_uptime.strftime("%s"))
     assert rebooted, "Device {} did not reboot".format(hostname)
 
+    # Verify that the links are all LCs are up
+    check_interfaces_and_services_all_LCs(duthosts, conn_graph_facts, xcvr_skip_list)
+
+    # Also make sure fanout hosts' links are up
+    link_status_on_all_LC(duthosts, localhost, fanouts_and_ports)
+
 
 def test_link_status_on_host_reboot(duthosts, localhost, enum_frontend_dut_hostname, 
                                     duts_running_config_facts, conn_graph_facts, 
                                     fanouthosts, xcvr_skip_list, tbinfo):
+    global MAX_TIME_TO_REBOOT
     duthost = duthosts[enum_frontend_dut_hostname]
     hostname = duthost.hostname
 
+    tc_name = os.environ.get('PYTEST_CURRENT_TEST').split(' ')[0]
+    plt_reboot_ctrl = get_plt_reboot_ctrl(duthost, tc_name, 'cold')
+    if plt_reboot_ctrl:
+        MAX_TIME_TO_REBOOT = plt_reboot_ctrl['wait']
+
```

Comment:
can you add `time.sleep(MAX_TIME_TO_REBOOT)` here

Reply:
With 'check_interfaces_and_services_all_LCs' at Line#176 as part of sup_reboot, do we still need this sleep here?

Reply:
then no need here